Utilities/File.h

```diff
@@ -728,14 +728,23 @@ namespace fs
 	}
 
 	template <typename... Args>
-	bool write_file(const std::string& path, bs_t<fs::open_mode> mode, const Args&... args)
+	bool write_file(const std::string& path, bs_t<fs::open_mode> mode, Args&&... args)
```

Comment:
Why Args&&? It's easier to work with const reference pack, plus no file writing methods can actually consume anything iirc.

Reply:
Fixed.